ubuntu

```diff
@@ -9,6 +9,22 @@ RUN apt-get update && \
     apt-get purge --autoremove -y && \
     rm -rf /var/lib/apt/lists/*
 
+ARG CMAKE_VERSION=3.17.3
```

Comment:
Wouldn't using https://apt.kitware.com/ be easier?

Reply:
This is how we do it in the main repo and it seems to be sufficient. :shrug:

Reply:
I tried but couldn't quite resolve package dependencies.